infra-examples/aws/k8s-cluster/README.rst

```diff
@@ -0,0 +1,90 @@
+Amazon Elastic Kubernetes Service (EKS)
```

Comment:
I'm not agains reStructuredText, but shouldn't we uniformly use markdown for documentation?